test/unit_test/ccb_test.cc

```diff
@@ -24,7 +24,7 @@ BOOST_AUTO_TEST_CASE(ccb_generate_interactions)
   CCB::calculate_and_insert_interactions(shared_ex, actions, interactions);
   std::sort(compare_set.begin(), compare_set.end());
   std::sort(interactions.begin(), interactions.end());
-  check_vectors(interactions, compare_set);
+  check_collections(interactions, compare_set);
```

Comment:
since we have a `check_collections_with_tolerance`, maybe rename to `check_collections_exact` for clarity?